user-interface/6004-CHAR-chart.md

```diff
@@ -0,0 +1,106 @@
+# Chart
+
+## Display options
+- **Must** be able to change time interval from a list of intervals ([1003-TRAN-001](#1003-TRAN-001)) :    
```

Comment:
should these not be 1000-CHAR-001?

Reply:
yeah, Will update all the approbation tagging once the actual ACs are all finialised just so that I have to change the numbers once etc.

Reply:
Fixed.